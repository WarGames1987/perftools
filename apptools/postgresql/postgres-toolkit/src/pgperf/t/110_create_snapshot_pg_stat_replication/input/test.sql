SELECT pgperf.create_snapshot_pg_stat_replication(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;

SELECT pgperf.create_snapshot_pg_stat_replication(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;

SELECT pgperf.create_snapshot_pg_stat_replication(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;

SELECT pgperf.delete_snapshot_pg_stat_replication(1);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;

SELECT pgperf.delete_snapshot_pg_stat_replication(2);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;

SELECT pgperf.delete_snapshot_pg_stat_replication(0);
SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
