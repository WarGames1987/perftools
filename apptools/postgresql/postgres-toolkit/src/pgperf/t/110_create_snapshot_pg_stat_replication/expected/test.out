SELECT pgperf.create_snapshot_pg_stat_replication(0);
 create_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

SELECT pgperf.create_snapshot_pg_stat_replication(1);
 create_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

SELECT pgperf.create_snapshot_pg_stat_replication(2);
 create_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_replication(1);
 delete_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_replication(2);
 delete_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

SELECT pgperf.delete_snapshot_pg_stat_replication(0);
 delete_snapshot_pg_stat_replication 
-------------------------------------
 t
(1 row)

SELECT count(*) FROM pgperf.snapshot_pg_stat_replication;
 count 
-------
     0
(1 row)

